packages/suite-desktop-core/e2e/tests/spawn-bridge.test.ts

```diff
@@ -51,4 +59,23 @@ testPlaywright.describe.serial('Bridge', () => {
             // ok
         }
     });
+
+    testPlaywright(
+        'App acquired device, EXTERNAL bridge is restarted, app reconnects',
+        async () => {
+            await TrezorUserEnvLink.startEmu({ wipe: true, version: '2-latest', model: 'T2T1' });
+            await TrezorUserEnvLink.setupEmu({});
```

Comment:
shoudln't it start bridge here? emu is probably not important

Reply:
Right, the test was passing since bridge was running from the previous test, so I changed stopBridge in beforeAll to beforeEach ot start each test the same way and then start it in this test.

The emu is required to "passThroughInitialRun" so we make sure device is connected before stopping bridge.